### ISSUE TITLE
Wrapper informs me I have outdated drivers on exit

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K-Infrastructure#1659]
When I close chobby I get a popup telling me about outdated drivers. Everything works fine for me so this popup should not occur. It will get annoying.

**Comment by Licho1** [comment]
This is really for engine devs .. wrapper just tells you this if engine reports this..

**Comment by Licho1** [comment]
You have one of these messages in your log:
No OpenGL drivers installed.
Please go to your GPU vendor's website and download their drivers.
Update your graphic-card driver!

I dont think i should be disabling this warning, you should instead update drivers!

**Comment by GoogleFrog** [comment]
Games should not ask users to upgrade their drivers when there is nothing wrong with the game.

**Comment by Anarchid** [comment]
> No OpenGL drivers installed.

Does Spring really say this if it can run OpenGL? If so, that's an engine bug.

**Comment by Licho1** [comment]
Yeah imo nothing we should do with the wrapper.. check out with engine devs why are you getting this message in logs..

**Comment by gajop** [comment]
Can you please post an infolog where it says that and everything still works normally?

**Comment by Licho1** [comment]
@GoogleFrog feedback required
